src/infrastructure/local-cache/adapters/web.ts

```
@@ -0,0 +1,32 @@
+    this.storage.setItem(key, value);
+  }
+
+  async removeItem(key: string) {
+    this.storage.removeItem(key);
+  }
+
+  async getKeys(startsWith: string): Promise<string[]> {
+    const keys: string[] = [];
+    for (let i = 0; i < this.storage.length; i++) {
+      const key = this.storage.key(i);
+      if (key && key.startsWith(startsWith)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)